.teamcity/settings.kts

```diff
@@ -142,8 +155,6 @@ fun Project.deployVersion() = BuildType {
     params {
         // enable editing of this configuration to set up things
         param("teamcity.ui.settings.readOnly", "false")
-        param("bintray-user", bintrayUserName)
```

Comment:
`settings.kts` is auto-generated by the infra plugin. Please run `./gradlew setupTeamCity --i` to make the infra plugin update the TeamCity configuration files. It should remove the deprecated bintray publishing params as well.